units/en/unit5/1_introduction.mdx

```diff
@@ -0,0 +1,85 @@
+# Introduction to Gradio for AI Agents
+
+<img src="https://huggingface.co/datasets/agents-course/course-images/resolve/main/en/bonus-gradio/whiteboard.jpg" alt="Gradio Module Planning"/>
+
+In the previous units, we learned how to create powerful AI Agents that can reason, plan, and take actions. But an Agent is only as effective as its ability to interact with users. This is where **Gradio** comes in.
+
+## Why Do We Need a UI for Our Agents?
+
+Meet Sarah, a data scientist who built an amazing AI Agent that can analyze financial data and generate reports. But there's a challenge:
+
+- Her teammates need to interact with the Agent
+- Not everyone is comfortable with code or command line
+- Users want to see the Agent's thought process
+- The Agent needs to handle file uploads and display visualizations
+
+The solution? A user-friendly interface that makes the Agent accessible to everyone.
+
+## What is Gradio?
+
+Gradio is a Python library that makes it easy to create **beautiful web interfaces** for your AI models and Agents. Think of it as a bridge between your Agent's capabilities and its users:
```

Comment:
```suggestion
[Gradio](https://github.com/gradio-app/gradio) is a Python library that makes it easy to create **beautiful web interfaces** for your AI models and Agents. Think of it as a bridge between your Agent's capabilities and its users:
```

Reply:
I like "web applications" instead of "web interfaces" personally.